build/run_all_tests

```diff
@@ -1,33 +1,36 @@
 #!/bin/sh
 
-test_dir=`dirname $0`/../out/Test.gn
+set -e
```

Comment:
Not sure if this is appropriate, but now the script requires to look at log output from all binaries to ensure everything went OK.